packages/core/src/blocks/defaultBlocks.ts

```diff
@@ -42,6 +42,7 @@ export const defaultBlockSpecs = {
   numberedListItem: NumberedListItem,
   checkListItem: CheckListItem,
   table: Table,
+  tableCell: TableCell,
```

Comment:
had a quick look as I was looking for the pm-view changes. Are you sure tablecell should be a separate block?

Reply:
You are right, shouldn't be. Was just fiddling with the types to see what it needed. Removed now